### ISSUE TITLE
prod - Issue with first survey - health conditions under a sub-set type

### ISSUE THREAD

**Issue by dbakumar** [episphere/connectApp#258]
The below issue raised by one of the study participant. 

health conditions under a sub-set type:
When you answer that you have been told by a doctor that you have that condition...it then moves to a screen where you can choose either how old you were when you were diagnosed, or the year of the diagnosis.   at this point - no response to entering answer and hitting the next button. 

a) tried changing from selecting the year, to inputting my age instead - no luck. 
b) chose the back button and started again, no luck.  
c) cannot continue past the point where  stuck.

**Comment by FrogGirl1123** [comment]
@naiyume Could you please first check that this isn't an issue with the PWA.  If that's eliminated as a cause we'll go back to the markup and the renderer.

**Comment by danielruss** [comment]
@dbakumar  I ran though module1 and said that i had Basil and Squamous cell skin cancer.  At the next question, I was able to set the year of diagnosis. Change to age, go back to year.  

Any chance we can get more info?  What question did (s)he run into problems?  Was it in the loop about the children?

**Comment by Davinkjohnson** [comment]
@danielruss when you set the year of diagnosis, were you able to continue to the next question? It seems like the issue states they could enter a value, but weren't able to continue once they did.

**Comment by danielruss** [comment]
yes I could continue.  It would be really useful if  we could reproduce his problem.

**Comment by FrogGirl1123** [comment]
Daniel are you testing in the renderer or in the PWA?  Since the issue happened in the PWA maybe it should be tested in the PWA in stage.  I really don't want to add test data to prod.

**Comment by danielruss** [comment]
I was working in DEV

**Comment by FrogGirl1123** [comment]
Thank you for sharing their feedback @dbakumar !

@danielruss do you want to see if you can recreate the issue or should be go ahead and close.  If you are still trying to recreate, I would suggest trying in stage since that environment is more like prod.

**Comment by anthonypetersen** [comment]
@FrogGirl1123 @danielruss are we still trying to re-create this or can this issue be closed?

**Comment by anthonypetersen** [comment]
@FrogGirl1123 @danielruss can this be closed?
